docs/wait.md

```diff
@@ -166,4 +174,38 @@ function sayThree()
 end
 ```
 
+Example (check if a die object is resting every 0.5 seconds):
```

Comment:
I'm not sure how I feel about this example. The idea with API docs is to try keep things simple, short and non-opinionated. There are multiple ways to achieve what you've done here. For example, you're using global variables (`isRestingID`) and functions (`startWatching`, `isResting`) which is something I'd personally advise against.

For an example, you want to keep it _real_ short. You could probably simplify this to:

> Example. Check if a die is resting every 0.5 seconds:
> 
> ```lua
> die.roll()
> local id 
> id = Wait.time(
>     function()
>         if die.isDestroyed() then
>             print("Die was deleted or entered a container")
>             Wait.stop(id) -- Stop future calls (repetitions) of this callback
>         elseif die.resting then
>             print("Die has come to rest")
>             Wait.stop(id) -- Stop future calls (repetitions) of this callback
>         end
>     end
>     0.5, -- 0.5 seconds
>     -1 -- Repeat indefinitely, or rather until we call Wait.stop(id)
> )
> ```

P.S. I've also corrected a common mistake, which is that people forget to check for destroyed objects, so you can end up with your callbacks being called indefinitely.

P.P.S. I'm using a new v13 function `die.isDestroyed()` instead of `die == nil`. I don't believe it's documented yet, but it should be easier for people to reason about!